www/src/components/shell/Terminal.js

```diff
@@ -3,13 +3,11 @@ import { useCallback, useContext, useEffect, useMemo, useRef, useState } from 'r
 import { XTerm } from 'xterm-for-react'
 import { FitAddon } from 'xterm-addon-fit'
 import { useQuery } from '@apollo/client'
-import { Button, Div, Flex, Span } from 'honorable'
-
-import { ReloadIcon } from 'pluralsh-design-system'
+import { Button, Div, Flex } from 'honorable'
```

Comment:
I think we should start importing `Button` from 'plural-sh-design-system' from now on since that version has added functionality over the default honorable one. Doesn't _really_ matter here, but may as well be consistent in case it ever diverges from the honorable one.